client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -1227,6 +1228,34 @@ public void testRollover() throws IOException {
         assertEquals(expectedParams, request.getParameters());
     }
 
+    public void testIndexPutSettings() throws IOException {
+        String[] indices = randomIndicesNames(0, 2);
+        UpdateSettingsRequest updateSettingsRequest = new UpdateSettingsRequest(indices);
```

Comment:
Can we also test the case where indices are not set at all, then the request would hold `null`?